ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt

```diff
@@ -7,6 +7,8 @@ Or type one of the following commands:
 /debug          - Toggle debug mode. You will see debug messages.
 /reset          - Reset REPL state.
                   All previous code snippets will be removed from REPL memory.
+/open <FILE>    - Load declarations defined in <FILE>. eg: /file my_file.bal
+/remove <NAMES> - Delete declarations specified by <NAMES>. eg: /file function1 var1 var2
```

Comment:
eg: => e.g.,

Reply:
How's the exact usage of these? `/open /file my_file.bal`?

Reply:
It is a typo, the example was supposed to say `/open my_file.bal`, so use the command as `/open my_file.bal` 
But the command is `/file my_file.bal` if using the last commit because I just changed that command to /file instead of /open as per @MaryamZi 's suggestion :)